src/components/Tag/Tag.tsx

```diff
@@ -72,7 +72,9 @@ const tagBaseStyles = ({ theme }: StyleProps) => css`
   label: tag;
```

Comment:
`tagClickableStyles` needs `text-align: left`, because the button will make the text centered.

Reply:
Ah, that's why it's needed 💡 

I had tested it with the base version and was confused because it was left-aligned by default.